### PR TITLE
Update routing

### PR DESCRIPTION
**Pull request by simonwhatley**
Tidying up the routes

See https://trello.com/c/D6jnXfbx/3065-refactor-prototype-routing

### CODE REVIEW

**Review by adamsilver — APPROVED** [review]
Should the application list live at /applications ?

File: app/routes/applications.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ module.exports = router => {
     const applicationId = req.params.applicationId
     const application = req.session.data.applications.find(app => app.id === applicationId)
 
-    res.render('application/index', {
+    res.render('applications/index', {
```
Comment (adamsilver) [inline app/routes/applications.js:8]: Can we make this the ‘show’ view?

File: app/routes/applications.js [hunk 2]
```diff
@@ -15,7 +15,7 @@ module.exports = router => {
     const applicationId = req.params.applicationId
     const application = req.session.data.applications.find(app => app.id === applicationId)
 
-    res.render('application/timeline', {
+    res.render('applications/timeline', {
```
Comment (adamsilver) [inline app/routes/applications.js:18]: you could do: `applications/timeline/show` but you don't have to.

File: app/routes/applications.js (hunk 1 above)
Comment (simonwhatley) [reply app/routes/applications.js:8]: Yep, I'll make the change.

File: app/routes/applications.js (hunk 2 above)
Comment (simonwhatley) [reply app/routes/applications.js:18]: Would this be `index` or `list`, rather than `show`, which would be reserved for single timeline events?

Comment (simonwhatley) [reply app/routes/applications.js]: The complexity of naming things. We'll go with the `show` convention for the time being.

**Pull request merged**